### PR TITLE
Better tsconfig

### PR DESCRIPTION
**Pull request by novemberborn**
See commits.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -7,14 +7,20 @@
 		"noImplicitThis": true,
 		"strictNullChecks": true,
 		"outDir": "_build/",
+		"baseUrl": "node_modules",
 		"removeComments": false,
 		"sourceMap": true,
 		"target": "es5",
-		"moduleResolution": "node"
+		"moduleResolution": "classic"
```
Comment (agubler) [inline tsconfig.json]: why?

Comment (novemberborn) [inline tsconfig.json]: My bad, `app` itself was outdated.

**Comment by novemberborn** [comment]
More changes :)

File: tsconfig.json [hunk 2]
```diff
@@ -1,20 +1,25 @@
 {
-	"version": "2.0.0",
+	"version": "2.0.3",
 	"compilerOptions": {
 		"declaration": false,
 		"module": "umd",
+		"moduleResolution": "node"
```
Comment (agubler) [inline tsconfig.json]: missing comma

Comment (novemberborn) [inline tsconfig.json]: Yea going to fast… fixed.

**Pull request merged**